src/cpp/rtps/reader/RTPSReader.cpp

```diff
@@ -367,6 +367,25 @@ uint64_t RTPSReader::get_unread_count() const
     return total_unread_;
 }
 
+uint64_t RTPSReader::get_unread_count(
+        bool mark_as_read)
+{
```

Comment:
Shouldn't the other overload call to this one with `false`

Reply:
The other overload is `const`, but this one isn't....